admin/class-automation.php

```diff
@@ -170,68 +162,43 @@ public static function get_automation_rules(): array {
 	}
 
 	/**
-	 * A render callback for the automation settings field.
+	 * A render callback for the REACT submenu page.
 	 */
-	public static function render_settings_field(): void {
-		$value = self::get_automation_rules();
-		// TODO: Dynamically get list of registered taxonomies.
-		?>
-			<fieldset>
-				<legend class="screen-reader-text">
-					<?php esc_html_e( 'Configuration', 'apple-news' ); ?> 1
-				</legend>
-				<div>
-					<label>
-						<?php esc_html_e( 'Taxonomy', 'apple-news' ); ?>
-						<select name="<?php echo esc_attr( self::OPTION_KEY ); ?>[0][taxonomy]">
-							<option value=""></option>
-							<option value="category" <?php selected( 'category' === ( $value[0]['taxonomy'] ?? '' ) ); ?>><?php esc_html_e( 'Category', 'apple-news' ); ?></option>
-							<option value="post_tag" <?php selected( 'post_tag' === ( $value[0]['taxonomy'] ?? '' ) ); ?>><?php esc_html_e( 'Tag', 'apple-news' ); ?></option>
-						</select>
-					</label>
-				</div>
-				<div>
-					<label>
-						<?php esc_html_e( 'Term ID', 'apple-news' ); ?>
-						<input name="<?php echo esc_attr( self::OPTION_KEY ); ?>[0][term_id]" type="number" value="<?php echo esc_attr( $value[0]['term_id'] ?? '' ); ?>" />
-					</label>
-				</div>
-				<div>
-					<label>
-						<?php esc_html_e( 'Field', 'apple-news' ); ?>
-						<select name="<?php echo esc_attr( self::OPTION_KEY ); ?>[0][field]">
-							<option value=""></option>
-							<?php foreach ( array_keys( self::FIELDS ) as $field_key ) : ?>
-								<option value="<?php echo esc_attr( $field_key ); ?>" <?php selected( ( $value[0]['field'] ?? '' ) === $field_key ); ?>><?php echo esc_html( $field_key ); ?></option>
-							<?php endforeach; ?>
-						</select>
-					</label>
-				</div>
-				<div>
-					<label>
-						<?php esc_html_e( 'Field Value', 'apple-news' ); ?>
-						<input name="<?php echo esc_attr( self::OPTION_KEY ); ?>[0][value]" type="text" value="<?php echo esc_attr( $value[0]['value'] ?? '' ); ?>" />
-					</label>
-				</div>
-			</fieldset>
-		<?php
+	public static function render_submenu_page(): void {
+		add_filter(
+			'should_load_block_editor_scripts_and_styles',
+			function() {
+				return true;
+			}
+		);
+		wp_add_iframed_editor_assets_html();
+		echo wp_kses(
+			'<div class="block-editor"><div class="editor-styles-wrapper"><div id="apple-news-options__page"></div></div></div>',
+			[ 'div' => [ 'id' => [] ] ]
+		);
```

Comment:
No need to `wp_kses` this because it's static HTML. You only need to run it through an escaping function if there's a variable in it.